### PR TITLE
Add flake8-sort, improve test ids, and verify requests

### PR DESCRIPTION
**Pull request by davehunt**
I've bundled a few changes here, but kept separate commits. I'd be happy to split into multiple pull requests if preferred.

* Added flake8-isort to check sort order of imports
* Include the locale name in the test id instead of `locale0`, `locale1`, etc
* Verify HTTP requests to avoid warnings
* Add local test results to `.gitignore`

### CODE REVIEW

**Comment by davehunt** [comment]
Passing adhoc builds for [staging](https://qa-preprod-master.fxtest.jenkins.stage.mozaws.net/job/bouncer.adhoc/16/) and [prod](https://qa-preprod-master.fxtest.jenkins.stage.mozaws.net/job/bouncer.adhoc/20/).

**Comment by davehunt** [comment]
@m8ttyB do you have some time to look over this?

File: tests/e2e/tests/test_redirects.py [hunk 1]
```diff
@@ -82,7 +83,7 @@ def test_that_checks_redirect_using_incorrect_query_values(self, base_url):
             self.response_info_failure_message(base_url, param, response)
 
     @pytest.mark.parametrize('os', _os)
-    @pytest.mark.parametrize('locale', _locales)
+    @pytest.mark.parametrize('locale', _locales, ids=lambda l: l.lang)
```
Comment (m8ttyB) [inline tests/e2e/tests/test_redirects.py]: Nice!

File: tests/e2e/tests/test_smoketests.py [hunk 2]
```diff
@@ -4,8 +4,8 @@
 
 import pytest
 
-from base import Base
-import releng_utils as utils
+from . import releng_utils as utils
+from .base import Base
```
Comment (m8ttyB) [inline tests/e2e/tests/test_smoketests.py]: +:100:

File: tests/e2e/setup.cfg [hunk 3]
```diff
@@ -0,0 +1,4 @@
+[isort]
```
Comment (m8ttyB) [inline tests/e2e/setup.cfg]: Interesting.

**Pull request merged**